ui/package.json

```diff
@@ -30,7 +30,7 @@
     "popper.js": "^1.16.1",
     "react": "^17.0.2",
     "react-bootstrap": "^2.3.0",
-    "react-bootstrap-table": "^4.3.1",
+    "react-bootstrap-table-next": "^4.0.3",
```

Comment:
I'm not sure react-bootstrap-table-next is compatible entirely with Boostrap 5, it only support Boostrap 4 at most ?